test/integration_tests/prototype/hifi_gan_pipeline_test.py

```diff
@@ -0,0 +1,65 @@
+import os
+import sys
+
+import pytest
+
+sys.path.append(os.path.join(os.path.dirname(__file__), "..", "..", "..", "examples"))
```

Comment:
@nateanl This makes me wonder if we should move sdr to library.

Reply:
yeah we should include `Si-SDR` into core. In terms of `SDR`, the implementation in `bss_eval` can deal with signal pairs that are not aligned (for example, if there are some samples shifted, the SDR score still works fine). We might want to consider that when implementing `SDR` metric.

Reply:
In the integration test I've switched to the SDR metric computed by `mir_eval.separation.bss_eval_sources` - this metric is less negative, but still negative, I haven't found the reason yet